app/Console/Commands/AchievementsCron.php

```diff
@@ -235,10 +235,14 @@ private function ForeverMember()
         foreach (User::all() as $user) {
             $forever = False;
             foreach (Committee::all() as $committee) {
-                $memberships = CommitteeMembership::withTrashed()->where('user_id', $user->id)->where('committee_id', $committee->id)->get();
+                $memberships = CommitteeMembership::withTrashed()
+                    ->where('user_id', $user->id)
+                    ->where('committee_id', $committee->id)
+                    ->with('committee')
```

Comment:
Not strictly necessary, but since it needs to be checked for all of the retrieved objects it should slightly improve performance to do eager loading on this relation.